google-cloud-tools-plugin/testSrc/com/google/gct/idea/appengine/cloud/AppEngineCloudConfigurableTest.java

```diff
@@ -155,7 +156,7 @@ private void initCloudConfigurable() {
   private File createTempFile() throws IOException {
     TemporaryFolder tempFolder = new TemporaryFolder();
     tempFolder.create();
-    File executable = new File(tempFolder.newFolder("bin"), "gcloud");
+    File executable = new File(tempFolder.newFolder("bin"), SystemInfo.isWindows ? "gcloud.cmd" : "gcloud");
```

Comment:
Sure I can do that. My thought process was that having the duplication in the unit tests asserts that we have those commands correct in the actual code. Kind of in the same way that its a good idea to duplicate properties from our message bundle in the unit tests rather than referring to them directly. What do you guys think?